docs/docs/dev_docs/contracts/portals/main.md

```diff
@@ -180,7 +189,7 @@ Designated callers are enforced at the contract level for contracts that are not
 
 - Non-native token (asset bridged from L1)
```

Comment:
Not sure if we should be calling it non-native while there still is a mint that could be without the bridge, but that might be me being pedantic 🤷.